.env.example

```diff
@@ -4,7 +4,7 @@ HIBISCUS_AWS_ACCESS_KEY_ID=REDACTED
 HIBISCUS_AWS_SECRET_ACCESS_KEY=REDACTED
 
 SSO_URL=PLACEHOLDER
```

Comment:
maybe put this one in the browser context i.e NEXT_PUBLIC_ as well if you feel like that's useful

Reply:
Ok yea decided to change it too just in case we need to use it client-side in the future